Tests/Perf/metrics-test.yml

```diff
@@ -11,4 +11,4 @@ startupTimeTest:
 
 binarySizeTest:
   diffMin: 200 KiB
-  diffMax: 700 KiB
+  diffMax: 900 KiB
```

Comment:
I would go for 800 so we get reminded again to do something against it.

Reply:
```suggestion
  diffMax: 800 KiB
```